### PR TITLE
Fix chaos timeout

### PR DESCRIPTION
**Pull request by MBkkt**
### Scope & Purpose

Backport of https://github.com/arangodb/arangodb/pull/18829

- [ ] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: js/client/modules/@arangodb/test-helper.js [hunk 1]
```diff
@@ -268,7 +268,8 @@ const buildCode = function(key, command, cn, duration) {
   let file = fs.getTempFile() + "-" + key;
   fs.write(file, `
 (function() {
-require('internal').SetGlobalExecutionDeadlineTo((${duration} + 10) * 1000);
```
Comment (dothebart) [inline js/client/modules/@arangodb/test-helper.js:271]: if you do this, you need to change the grace period of the watching arangosh as well.

Comment (MBkkt) [reply js/client/modules/@arangodb/test-helper.js:271]: It's backport

Comment (dothebart) [reply js/client/modules/@arangodb/test-helper.js:271]: please create a subsequent PR then. the result aggregation behaviour is probably better if the subshells terminate on their own.

Comment (Dronplane) [reply js/client/modules/@arangodb/test-helper.js:271]: @dothebart  Could you please elaborate a bit more:
1. What and where should we change more?
2. Is it related to this backport only or we need to do something in devel too?
Or if it is not much effort, could you please just push necessary changes here? (And open a devel PR if necessary). 
As for now these changes seems to be working in devel and we don't see targeted errors anymore. But maybe we are missing something.

Comment (dothebart) [reply js/client/modules/@arangodb/test-helper.js:271]: If I remember this correctly the deadline of the spawned arangoshs is calculated against a timeout in the testing.js arangosh by the specified `duration`. If you increase the sluggishness of the deadline to the duration, you need to adjust that in the calling suite as well.

Comment (Dronplane) [reply js/client/modules/@arangodb/test-helper.js:271]: As I see, after clients started
`internal.sleep(duration);` is done e.g. no adjustments (+10s) are made. So I assume still nothing is needed. Or I`m missing something?

Comment (Dronplane) [reply js/client/modules/@arangodb/test-helper.js:271]: duration - is expected arangosh instance run.
Deadline - is this duration + some extent to accommodate possible delay

Comment (dothebart) [reply js/client/modules/@arangodb/test-helper.js:271]: ok, so if the time is too short, why then not increase the digit on the outside, but somewhere inside?

**Review by KVS85 — APPROVED** [review]

**Pull request merged**